src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHost.cs

```diff
@@ -48,7 +48,6 @@ public sealed class WebAssemblyHost : IAsyncDisposable
             string? persistedState)
         {
             // To ensure JS-invoked methods don't get linked out, have a reference to their enclosing types
-            GC.KeepAlive(typeof(EntrypointInvoker));
```

Comment:
FYI @eerhardt

Reply:
This should also resolve https://github.com/dotnet/aspnetcore/issues/31148 since it is getting rid of the last call to `Console` in a default Blazor WASM app.